frontend/src/pages/Landing/index.js

```diff
@@ -303,7 +303,6 @@ function Landing() {
               filters={filters}
               onApplyFilters={onApply}
               onRemoveFilter={onRemoveFilter}
-              // filterConfig={getFilters()}
```

Comment:
Can we remove this? Or will it come into play later?

Reply:
this line is me removing it